cluster-service/deploy/templates/deployment.yaml

```
@@ -136,6 +136,13 @@ spec:
         - name: azure-operators-managed-identities-config
           mountPath: /configs/azure-operators-managed-identities-config.yaml
           subPath: azure-operators-managed-identities-config.yaml
+        {{- if .Values.tracing.enabled }}
```

```suggestion
        {{- if ne .Values.Tracing.Address "" }}
```

Maybe we want to replace the `enabled` field with a check for an empty address?

---

Sounds good to me but maybe we should rather check if  `tracing.exporter` is empty or not?
Having a `tracing.enabled ` bool flag feels redundant and error-prone (e.g. what happens if you set enabled=true but no endpoint?)